src/Models/SiteUpstream.php

```diff
@@ -55,12 +56,22 @@ protected function parseAttributes($data)
         }
         return $data;
     }
-    
+
      /**
      * @return string[]
      */
     public function getReferences()
     {
         return [$this->id, $this->get('product_id'), $this->get('label'), $this->get('machine_name'),];
     }
+
+    /**
+     * Returns TRUE if is the "drops-8" upstream.
+     *
+     * @return bool
+     */
+    public function isDrops8Upstream(): bool
```

Comment:
Used here https://github.com/pantheon-systems/terminus-conversion-tools-plugin/blob/main/src/Commands/ConvertToComposerSiteCommand.php#L37

Reply:
I don't think that Terminus should be in the business of knowing upstream machine names. Move this over to the conversion tools plugin. Rather than having a class with a lot of is-this-or-that-upstream, I think we could just define constants (e.g. public class constants) to compare against. I care 1 out of 10, though, so you can keep the implementation as "is" methods if you really feel that is better.

Reply:
removed